disco_aws_automation/disco_dynamodb.py

```diff
@@ -132,3 +141,153 @@ def _convert_table_to_dict(table):
                 "StreamSpecification": table.stream_specification,
                 "LatestStreamLabel": table.latest_stream_label,
                 "LatestStreamArn": table.latest_stream_arn}
+
+
+class AsiaqDynamoDbBackupManager(object):
+    """
+    Management tool for creating, restoring from, and inspecting DynamoDB table backups.
+    """
+    S3_BUCKET_KEY = "dynamodb-backup"
+    BACKUP_DESC_TEMPLATE = "Periodic backup for %s table in %s env"
+
+    BACKUP_PIPELINE_TEMPLATE = 'dynamodb_backup'
+    RESTORE_PIPELINE_TEMPLATE = 'dynamodb_restore'
```

Comment:
Are these template files going to exist in each team's own config repo or are we mandating that users have to use the asiqa_astro_config? If it's the former, we could potentially have different versions of the template and inadvertently changing the pipeline.